### PR TITLE
feat: add output flag to specify output dir

### PR DESCRIPTION
**Pull request by easonlin404**
address #257 #251

### CODE REVIEW

**Comment by easonlin404** [comment]
This PR will also remove`--swagger` flag, using '--output' flag to unify output files output dir.

@pei0804

**Comment by pei0804** [comment]
👀

File: gen/gen.go [hunk 1]
```diff
@@ -23,17 +23,32 @@ func New() *Gen {
 	return &Gen{}
 }
 
+// Config presents Gen configurations.
+type Config struct {
```
Comment (pei0804) [inline gen/gen.go]: 👍

**Comment by pei0804** [comment]
This section change often so I delete it.
https://github.com/swaggo/swag#swag-cli

File: gen/gen_test.go [hunk 2]
```diff
@@ -10,54 +10,110 @@ import (
...
+		t.Fatal(err)
+	}
+	if _, err := os.Stat(path.Join("../testdata/simple/docs", "swagger.json")); os.IsNotExist(err) {
+		t.Fatal(err)
+	}
+	if _, err := os.Stat(path.Join("../testdata/simple/docs", "swagger.yaml")); os.IsNotExist(err) {
+		t.Fatal(err)
+	}
+
+	//TODO: remove gen files
```
Comment (pei0804) [inline gen/gen_test.go]: `outputdir` has been replace so it's ok now.

**Pull request merged**